### PR TITLE
debian 8 cleanup

### PR DESCRIPTION
**Pull request by jburel**
Fix typo and remove extra lines

### CODE REVIEW

File: omero-installation/omero_installation_debian8_ice35.md [hunk 1]
```diff
@@ -152,7 +152,4 @@ You can now start the OMERO.server and OMERO.web manually:
 	su - omero -c "OMERO.server/bin/omero web start"
 
 To start OMERO.server and OMERO.web automatically, please read
-[linux walkthrough/Running OMERO](https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/server-linux-walkthrough.html)
-
-
-
+[linux walkthrough/Running OMERO](https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/server-linux-walkthrough.html)
```
Comment (mtbc) [inline omero-installation/omero_installation_debian8_ice35.md:155]: guess we'll need some systemd service files someday!

**Comment by mtbc** [comment]
Good to merge.

File: omero-installation/omero_installation_debian8_ice35.md (hunk 1 above)
Comment (jburel) [reply omero-installation/omero_installation_debian8_ice35.md:155]: they are part of the main doc

Comment (mtbc) [reply omero-installation/omero_installation_debian8_ice35.md:155]: ah, maybe someday could link from https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/server-linux-walkthrough.html#running-omero

**Pull request merged**